### PR TITLE
Remove degraded

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: pkg/operator2/starter.go [hunk 1]
```diff
@@ -156,6 +157,16 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	staleConditions := staleconditions.NewRemoveStaleConditions(
+		[]string{
+			// in 4.1.0 this was accidentally in the list.  This can be removed in 4.3.
+			"Degraded",
```
Comment (enj) [inline pkg/operator2/starter.go]: Use the const.

File: pkg/operator2/starter.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package operator2
 
 import (
 	"fmt"
+	"github.com/openshift/library-go/pkg/operator/staleconditions"
```
Comment (enj) [inline pkg/operator2/starter.go]: move down.

File: pkg/operator2/starter.go (hunk 1 above)
Comment (deads2k) [reply pkg/operator2/starter.go]: > Use the const.

? the const should be deleted.  It's a const that shouldn't exist because any usage of it is wrong.

**Comment by deads2k** [comment]
if https://github.com/openshift/cluster-kube-scheduler-operator/pull/159 works, then I think this is ready.

File: pkg/operator2/starter.go (hunk 1 above)
Comment (enj) [reply pkg/operator2/starter.go]: Please follow-up and actually delete the const.

**Comment by enj** [comment]
#159

/close

**Pull request closed without merging**